dquartic/model/model.py

```diff
@@ -268,12 +269,12 @@ def p_sample(self, x_t, t, init_cond=None, attn_cond=None):
 
         if self.pred_type == "eps":
             # Predict noise
-            eps_pred = self.model(x_t, t_tensor, init_cond, attn_cond)
+            eps_pred = self.model(x_t, t_tensor, init_cond, attn_cond, precursor_cond)
```

Comment:
The current backbone model is the unet1d model, which the forward method currently accepts [`forward(self, x, time, init_cond=None, attn_cond=None)`](https://github.com/Roestlab/diffusion-deconvolution-dia-msms-data/blob/801a5108d57a8dcfa92456ee5ea8af716f1c1e96/dquartic/model/unet1d.py#L1086).